### PR TITLE
SITES-1757: extract out core component test models & utils to separat…

### PR DESCRIPTION
**Pull request by dhavkuma**
…e folder

| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | SITES-1757 
| Patch: Bug Fix?          | No
| Minor: New Feature?      | Refactor
| Major: Breaking Change?  | No
| Tests Added + Pass?      | NA
| Documentation Provided   | Not required (code comments and or markdown)
| Any Dependency Changes?  | No
| License                  | Apache License, Version 2.0

Moved core component test models and util classes out of testing/it/e2e-selenium to testing/it/e2e-selenium-utils.
This will help reuse these test models and util classes in  other repos with ui-tests based on selenium jupiter such as RTE

### CODE REVIEW

**Comment by dhavkuma** [comment]
Please review @absarasw @vladbailescu

File: testing/it/e2e-selenium-utils/src/main/java/com/adobe/cq/wcm/core/components/it/seljup/constant/CoreComponentConstants.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.it.seljup.constant;
+
+/**
+ * This includes the constants used over entire module.
+ */
+public class CoreComponentConstants {
```
Comment (absarasw) [inline testing/it/e2e-selenium-utils/src/main/java/com/adobe/cq/wcm/core/components/it/seljup/constant/CoreComponentConstants.java:21]: Should this be change to just Constants or something else?

**Review by absarasw — CHANGES_REQUESTED** [review]

File: testing/it/e2e-selenium-utils/pom.xml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        <version>2.17.11-SNAPSHOT</version>
+        <relativePath>../../../parent/pom.xml</relativePath>
+    </parent>
+
+
+    <!-- ====================================================================== -->
+    <!-- P R O J E C T   D E S C R I P T I O N                                  -->
+    <!-- ====================================================================== -->
+    <groupId>com.adobe.cq</groupId>
+    <artifactId>core.wcm.components.it.utils.e2e-selenium</artifactId>
```
[the remark this answers is not in the document]
Comment (dhavkuma) [reply testing/it/e2e-selenium-utils/pom.xml]: Done

File: testing/it/e2e-selenium-utils/src/main/java/com/adobe/cq/wcm/core/components/it/seljup/constant/CoreComponentConstants.java (hunk 1 above)
Comment (dhavkuma) [reply testing/it/e2e-selenium-utils/src/main/java/com/adobe/cq/wcm/core/components/it/seljup/constant/CoreComponentConstants.java]: Done. Changed to RequestConstants.java

**Review by vladbailescu — CHANGES_REQUESTED** [review]
LGTM except for what was noted by @absarasw, the README file should provide more details on how these are supposed to be used.

**Comment by dhavkuma** [comment]
@vladbailescu I have updated the readme file. @absarasw Please check.

File: testing/it/e2e-selenium-utils/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Purpose
+
+This is Model and Utility classes for WCM Core Components UI testing.
+
+# Requirement
+
+## Java version
+Use JDK 11 to compile
```
[the remark this answers is not in the document]
Comment (dhavkuma) [reply testing/it/e2e-selenium-utils/README.md]: Done

**Comment by dhavkuma** [comment]
@vladbailescu  @msagolj  Can you please restart the pipeline for this.
cc @absarasw

**Comment by dhavkuma** [comment]
Closing this PR due to old build failed before PR ammend. Will open a new PR.

**Comment by dhavkuma** [comment]
Opened https://github.com/adobe/aem-core-wcm-components/pull/1831

**Pull request closed without merging**